### ISSUE TITLE
normalize & get_feature_names are depreciating

### ISSUE THREAD

**Issue by BryanSWeber** [microsoft/EconML#605]
I'm running through the examples on page:
https://github.com/microsoft/EconML/blob/main/notebooks/OrthoIV%20and%20DRIV%20Examples.ipynb

It looks like several elements are getting phased out of a dependency within econML - which could be a problem in the future, some have already depreciated.

Execution of cells 1-6 returns the error/warning in cell 6:

`'normalize' was deprecated in version 1.0 and will be removed in 1.2. Please leave the normalize parameter to its default value to silence this warning. The default behavior of this estimator is to not do any normalization. If normalization is needed please use sklearn.preprocessing.StandardScaler instead.`

`Function get_feature_names is deprecated; get_feature_names is deprecated in 1.0 and will be removed in 1.2. Please use get_feature_names_out instead.`
